gatsby-config.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
 				pathToConfigModule: 'src/utils/typography.js'
 			}
 		},
+		'gatsby-plugin-less',
```

Comment:
If we want to support resolution independence in the enact-docs (do we?), then we need to alter this to include the postcss plugin for it.  If we want 1-to-1 consistency with CLI, we may want to have specific [`postCssPlugins`](https://www.gatsbyjs.org/packages/gatsby-plugin-less/#postcss-plugins) to match the [identical specific ones in CLI](https://github.com/enactjs/cli/blob/master/config/webpack.config.js#L83).  That said, we don't necessarily need 1-to-1 if it's not desired and would prefer to the approach this the Gatsby way.